examples/hosts/app-integration/external-controller/src/app.ts

```diff
@@ -63,9 +39,17 @@ async function start(): Promise<void> {
 
     // Get or create the document depending if we are running through the create new flow
 
-    const client = new FrsClient(connectionConfig);
+    const client = new TinyliciousClient();
+
+    let attach: undefined | (() => Promise<void>);
+
+    const customAttach = async (container, a) => {
+        // perform any customAttach actions here
+        attach = a;
```

Comment:
Wouldn't this also potentially allow the developer to do the opposite and modify `a`? i.e. if they did something like,
```
const customAttach = async (container, a) => {
        a = () => console.log("foo");
};
```
It might be a little confusing for them that the first `container` parameter is one that they should modify, and the second `a` parameter is one that they should just set to their own variable.

Could we consider passing in a `customAttach` function that just takes the `container` and then returning an attach callback from `createContainer`, perhaps in `containerServices`? And then they can call that callback on the `attachButton` click